src/interfacers/EmonHubMinimalModbusInterfacer.py

```diff
@@ -245,6 +299,9 @@ def set(self, **kwargs):
                     if 'address' in setting[meter]:
                         address = int(setting[meter]['address'])
                         self._log.info("Setting %s meters %s address %s", self.name, meter, address)
+                    
+                    if 'type' in setting[meter]:
```

Comment:
Is this backward compatible?

Reply:
heres the elif statements after the type lookups
'''
elif self.datatype == 'int':
                                    value = self._rs485.read_register(int(self._settings['meters'][meter]['registers'][i]), functioncode=3)
                                elif self.datatype == 'float':
                                    value = self._rs485.read_float(int(self._settings['meters'][meter]['registers'][i]), functioncode=4, number_of_registers=2)
'''